src/main/java/org/spongepowered/api/command/args/GenericArguments.java

```diff
@@ -120,8 +122,14 @@ public static CommandElement none() {
         return NONE;
     }
 
-    static CommandElement markTrue(String flag) {
-        return new MarkTrueCommandElement(flag);
+    /**
+     * Expects no arguments. Adds 'true' to the context when parsed.
+     *
+     * @param key the key to store 'true' under
+     * @return the argument
+     */
+    public static CommandElement markTrue(Text key) {
```

Comment:
I wanted it personally because I was re-implementing flags and had to copy this class because it was under default access. I would much rather refactor this into an element that inputs a given constant value when parsed as I think there's more use in that (and it's pretty simple to do).